attestation/src/measurement.rs

```diff
@@ -204,10 +251,38 @@ impl<'a, D: Digest> AttestationManager<D> {
         extn.encode_to_slice(extn_buf)
             .map_err(Error::InvalidExtensionDer)
     }
-}
 
-impl<D: Digest> Default for AttestationManager<D> {
-    fn default() -> Self {
-        Self::new(ID_SHA_384)
+    /// Generate an asymetric key pair from the attestation CDI.
+    /// This key pair will be used for signing certificates requests coming from TVMs.
+    #[allow(clippy::needless_borrow)]
+    pub fn key_pair(&self) -> Result<Keypair> {
+        // From the OpenDice implementation
+        let asym_salt: [u8; 64] = [
+            0x63, 0xB6, 0xA0, 0x4D, 0x2C, 0x07, 0x7F, 0xC1, 0x0F, 0x63, 0x9F, 0x21, 0xDA, 0x79,
+            0x38, 0x44, 0x35, 0x6C, 0xC2, 0xB0, 0xB4, 0x41, 0xB3, 0xA7, 0x71, 0x24, 0x03, 0x5C,
+            0x03, 0xF8, 0xE1, 0xBE, 0x60, 0x35, 0xD3, 0x1F, 0x28, 0x28, 0x21, 0xA7, 0x45, 0x0A,
+            0x02, 0x22, 0x2A, 0xB1, 0xB3, 0xCF, 0xF1, 0x67, 0x9B, 0x05, 0xAB, 0x1C, 0xA5, 0xD1,
+            0xAF, 0xFB, 0x78, 0x9C, 0xCD, 0x2B, 0x0B, 0x3B,
```

Comment:
If this a an immutable const array, perhaps it doesn't need to be allocated on the stack?